create_table.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE users (
```

Comment:
This file has now been deleted. Gabriel has set up a script to create the database